### PR TITLE
addresses issue #37

### PR DESCRIPTION
**Pull request by kachnov**

### CODE REVIEW

**Comment by kyrahabattoir** [comment]
I don't want to nitpick but can you remove the files that haven't been touched?

**Comment by mattroks101** [comment]
What Kyrah said.

File: code/modules/ZAS/Fire.dm [hunk 1]
```diff
@@ -181,6 +181,7 @@ turf/proc/hotspot_expose(exposed_temperature, exposed_volume, soh = 0)
 	animate(src, color = fire_color(air_contents.temperature), 5)
 	set_light(l_color = color)
 	playsound(src, 'sound/effects/fire_loop.ogg', 100, 0)
+
```
Comment (kachnov) [inline code/modules/ZAS/Fire.dm]: I think this extra space is necessary

**Comment by kachnov** [comment]
done

**Pull request merged**